plugins/wallabag.koplugin/main.lua

```diff
@@ -260,6 +261,17 @@ function Wallabag:addToMainMenu(menu_items)
                                     )
                                 end,
                             },
+                            {
+                                text = _("Download only starred articles"),
```

Comment:
For consistency with a few lines up.
```suggestion
                                text = _("Only download starred articles"),
```